client/components/TestReview/queries.js

```diff
@@ -1,15 +1,22 @@
 import { gql } from '@apollo/client';
 import {
   AT_FIELDS,
-  TEST_PLAN_VERSION_FIELDS
+  TEST_PLAN_VERSION_FIELDS,
+  ISSUE_FIELDS
 } from '@components/common/fragments';
 
 export const TEST_REVIEW_PAGE_QUERY = gql`
   ${AT_FIELDS}
   ${TEST_PLAN_VERSION_FIELDS}
+  ${ISSUE_FIELDS('all')}
   query TestReviewPageQuery($testPlanVersionId: ID!) {
     testPlanVersion(id: $testPlanVersionId) {
       ...TestPlanVersionFields
+      testPlan {
+        issues {
+          ...IssueFieldsAll
+        }
```

Comment:
```suggestion
        issues {
          ...IssueFieldsAll
          at {
            ...AtFields
          }
        }
```

Otherwise the 'Assistive Technology' column in the review page's GitHub Issues table defaults to 'AT not specified'